doc/classes/ClassDB.xml

```diff
@@ -200,7 +200,8 @@
 		<method name="get_class_list" qualifiers="const">
 			<return type="PackedStringArray" />
 			<description>
-				Returns the names of all the classes available.
+				Returns the names of all available built-in classes.
+				See also [method ProjectSettings.get_global_class_list] to get all registered global classes.
```

Comment:
Maybe, not sure.
```suggestion
				[b]Note:[/b] To get all registered global classes, use [method ProjectSettings.get_global_class_list] instead.
```

Reply:
Yeah, but in this case there is no obvious correlation between these two methods. In fact they do completely different things, so to avoid any possible ambiguity it's probably better to be explicit.

It's not like many of String's methods that are very similar from one another.

Reply:
I applied the suggestion you made. I'm not sure if you were also implying the other cross link should be changed to a note as well.